service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -61,6 +79,70 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
         return respond(caseDetails);
     }
 
+    @PostMapping("/submitted")
+    public void handleSubmitted(@RequestBody CallbackRequest callbackRequest) {
+        final CaseData caseData = getCaseData(callbackRequest);
+        CaseDetails caseDetails = callbackRequest.getCaseDetails();
+
+        String migrationId = (String) caseDetails.getData().get(MIGRATION_ID_KEY);
+
+        if ("DFPL-702".equals(migrationId)) {
```

Comment:
we need to remove this, the field won't be present on the case data during the submitted callback, we should always perform this supplementary data update I think (and we will make sure that when this is merged in NO ONE performs migrations until we remove it again the day after release)